src/cmd-fetch

```diff
@@ -85,31 +100,47 @@ fi
 
 prepare_build
 
-args=
+lock_args=
+extra_args=
+
+# Apply autolock from another build for this version if no base lockfile
+# exists. Do this before so that overrides come after.
+if [ ! -f "${manifest_lock}" ] && [ -n "${AUTOLOCK_VERSION}" ]; then
```

Comment:
Since the [proposed pipeline code](https://github.com/coreos/fedora-coreos-pipeline/pull/794/files) will only pass in `--autolock=VERSION` on mechanical streams that have no base lockfile I propose that we error out here if `--autolock=VERSION` was passed and a base lockfile already exists.